src/Images.jsx

```diff
@@ -73,18 +73,30 @@ class Images extends React.Component {
         }));
     }
 
-    handleRemoveImage() {
+    handleRemoveImage(tags, all) {
         const image = this.state.imageWillDelete.Id;
         this.setState({
             selectImageDeleteModal: false,
         });
-        client.delImage(this.state.imageWillDelete.isSystem, image, false)
-                .catch(ex => {
-                    this.imageRemoveErrorMsg = ex.message;
-                    this.setState({
-                        setImageRemoveErrorModal: true,
+        if (all)
+            client.delImage(this.state.imageWillDelete.isSystem, image, false)
+                    .catch(ex => {
+                        this.imageRemoveErrorMsg = ex.message;
+                        this.setState({
+                            setImageRemoveErrorModal: true,
+                        });
                     });
-                });
+        else {
+            // Call another untag once previous one resolved. Calling all at once can result in undefined behavior
+            const tag = tags.shift();
+            const i = tag.lastIndexOf(":");
+            client.untagImage(this.state.imageWillDelete.isSystem, image, tag.substring(0, i), tag.substring(i + 1, tag.length))
+                    .then(() => {
+                        if (tags.length > 0)
+                            this.handleRemoveImage(tags, all);
+                    })
+                    .catch(console.log);
```

Comment:
I know we do that everywhere in podman but if untagImage fails this won't show an error in the UI.

Reply:
Yeah, I want to address all of those in some other PR. We do this rather often and it is not good. Thought some global alert handler would be great, thus just `console.log` for now.

Reply:
Sure - I am fine with that.